zenodo/config.py

```diff
@@ -744,7 +748,8 @@ def _(x):
 #: Default database host.
 SQLALCHEMY_DATABASE_URI = os.environ.get(
     "SQLALCHEMY_DATABASE_URI",
-    "postgresql+psycopg2://localhost/zenodo")
+    #  "postgresql+psycopg2://localhost/zenodo")
+    'sqlite:///pippo.db')
```

Comment:
ops... this and the other one (the token) was only to play in my local env... XD
Inadvertently I added to the PR.
I'll remove this changes!